stream.go

```diff
@@ -155,6 +155,15 @@ func (cs *clientStream) Trailer() metadata.MD {
 }
 
 func (cs *clientStream) SendMsg(m interface{}) (err error) {
+	if EnableTracing {
+		cs.traceInfo.tr.LazyLog(payload{m}, true)
```

Comment:
I don't think the particular use of payloads for streaming should matter. Unary RPCs can be just as big. Controls over the size are a fine thing to have, but that's independent of this change.

(There's no need for control on the number of payloads; the trace package will only hold on to 10 events per trace. The only constraint may be on a per-payload basis, which will be a little harder to do since we permit arbitrary codecs. I have ideas for how to handle that though.)